app_flutter/lib/widgets/pulse.dart

```diff
@@ -0,0 +1,73 @@
+// Copyright (c) 2019 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:math' as math;
+
+import 'package:flutter/widgets.dart';
+
+class Pulse extends StatefulWidget {
```

Comment:
Can you add a quick dart doc for explaining that is used to pulsate tasks that are currently running in the TaskGrid?

Reply:
yup! Sorry I keep forgetting to add dartdocs to the top-level classes!